Resources/Locale/en-US/interaction/interaction-popup-component.ftl

```diff
@@ -0,0 +1,32 @@
+### Interaction Popup component
+
+## Petting animals
+
+petting-success-generic = You pet {THE($target)} on {POSS-ADJ($target)} head.
+petting-success-soft-floofy = You pet {THE($target)} on {POSS-ADJ($target)} soft floofy head.
+
+petting-success-bird = You pet {THE($target)} on {POSS-ADJ($target)} cute feathery head.
+petting-success-cat = You pet {THE($target)} on {POSS-ADJ($target)} fuzzy little head.
+petting-success-crab = You pet {THE($target)} on {POSS-ADJ($target)} smooth little head.
+petting-success-dog = You pet {THE($target)} on {POSS-ADJ($target)} soft floofy head.
+petting-success-frog = You pet {THE($target)} on {POSS-ADJ($target)} slippery little head.
+petting-success-goat = You pet {THE($target)} on {POSS-ADJ($target)} horned floofy head.
+petting-success-goose = Against all odds, you manage to pet {THE($target)} on {POSS-ADJ($target)} horrible little head.
+petting-success-reptile = You pet {THE($target)} on {POSS-ADJ($target)} scaly little head.
+petting-success-sloth = You pet {THE($target)} on {POSS-ADJ($target)} slow moving head.
+petting-success-space-cat = You pet {THE($target)} on {POSS-ADJ($target)} glass domed head.
+petting-success-tarantula = You pet {THE($target)} on {POSS-ADJ($target)} hairy little head.
+
+
+petting-failure-generic = You reach out to pet {THE($target)}, but {SUBJECT($target)} {CONJUGATE-BE($target)} aloof towards you.
+
+petting-failure-bat = You reach out to pet {THE($target)}, but {SUBJECT($target)} {CONJUGATE-BE($target)} too hard to catch!
+petting-failure-crab = You reach out to pet {THE($target)}, but {SUBJECT($target)} snaps its claws in your general direction!
+petting-failure-goat = You reach out to pet {THE($target)}, but {SUBJECT($target)} stubbornly refuses!
+petting-failure-goose = You reach out to pet {THE($target)}, but {SUBJECT($target)} {CONJUGATE-BE($target)} too horrible!
+petting-failure-sloth = You reach out to pet {THE($target)}, but {SUBJECT($target)} somehow dodge with ludicrous speed!
```

Comment:
```suggestion
petting-failure-sloth = You reach out to pet {THE($target)}, but {SUBJECT($target)} somehow dodges with ludicrous speed!
```

Reply:
I gave sloth epicene by default so I believe original is correct:

"You reach out to pet the sloth, but they somehow dodge with ludicrous speed!"

Reply:
Wasn't sure if I could write a custom CONJUGATE-DODGE function or something within this ftl. Probably wouldn't belong anywhere central because it's so specific to this one.

Is there a way I can make this more robust? Or do we think it's fine for now because the default makes an edge case rare?

Reply:
uhhhhhhhhhh we could just have a generic function for conjugating simple verbs, basically just adding an 's' on the end for he/she/it. but thats out of scope, you can leave it as 'dodge' for now

Reply:
actually ill just do it myself

Reply:
That'd be awesome, thanks! Same case with "refuse/refuses" under petting-failure-goat. I'll do a full check for any others when I fix those.

Reply:
Will handle this small fix in a future PR when such a function is made. 😄